scripts/seed.js

```diff
@@ -51,14 +51,27 @@ async function createBodies() {
         'Other'
     ];
 
+    const codes = [
+        'ANT',
+        'COA',
+        'CON',
+        'INT',
+        'WRK',
+        'COS',
+        'COT',
+        'PRO',
+        'PAR',
+        'OTH'
+    ];
+
     const bodies = [];
 
-    for (const type of types) {
-        const typeLowerCase = type.toLowerCase();
+    for (let i = 0; i < types.length; i++) {
```

Comment:
Like I mentioned in the call, a forEach loop could also be done. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
But this also works fine